docs/src/modules/components/DemoSandbox.js

```diff
@@ -128,19 +129,24 @@ function getTheme(outerTheme) {
     outerTheme.palette.primary?.main &&
     outerTheme.palette.primary.main !== brandingDesignTokens.palette.primary.main;
   const resultTheme = createTheme(
-    {
-      palette: {
-        mode: outerTheme.palette.mode || 'light',
-        ...(isCustomized && {
-          // Apply color from the color playground
-          primary: { main: outerTheme.palette.primary.main },
-          secondary: { main: outerTheme.palette.secondary.main },
-        }),
+    // merging in the outer theme to preserve custom tokens added by varnish.
+    // this may have larger consequences yet to be understood... or it could be fine.
```

Comment:
I don't really follow what we're doing here, so I can't comment on the repercussions :)